app/controllers/application_controller.rb

```diff
@@ -1,2 +1,12 @@
 class ApplicationController < ActionController::Base
+    helper_method :current_user
+    helper_method :logged_in?
```

Comment:
@idprietop what a helper_method is? and what is it used for?

Reply:
helper_mehod is a helper method which allows us to explicitly share some methods defined in the controller so that they are available in the view.

Reply:
@idprietop good, what makes different a helper method defined in the Application controller?

Reply:
What makes it different to define helper methods in the application controller is that the other controllers will have access to these methods because they inherit from this controller so it will not be necessary to define them in each of the controllers.